packages/toast/src/use-toast.tsx

```diff
@@ -53,7 +53,7 @@ export interface UseToastOptions {
   /**
    * The alert component `variant` to use
    */
-  variant?: string
+  variant?: "subtle"| "solid"| "left-accent" | "top-accent"
```

Comment:
This is quite strict and assumes that the user doesn't customize the theme to include more variants.

```suggestion
  variant?: "subtle"| "solid"| "left-accent" | "top-accent" | (string & {})
```

Use this walkaround to allow custom variant values.